packages/presentation/presentation_containers/interfaces/presentation_container.ts

```diff
@@ -11,11 +11,15 @@ import { PublishesLastSavedState } from './last_saved_state';
 
 export interface PanelPackage {
   panelType: string;
-  initialState: unknown;
+  initialState?: object;
 }
 
 export type PresentationContainer = Partial<PublishesViewMode> &
   PublishesLastSavedState & {
+    addNewPanel: <ApiType extends unknown = unknown>(
+      panel: PanelPackage,
+      silent?: boolean
```

Comment:
I actually purposefully went more crypic with this one, because the presentation container interface could be fulfilled with any implementation, and they may not use a `toast` or anything of the sort.

Reply:
How about `displaySuccess`? Its vague on how success is displayed but conveys what this value toogles.

Reply:
Sure that works!

Reply:
done in [192fd05](https://github.com/elastic/kibana/pull/176110/commits/192fd054c24f0ac05c0ef107e9417781fc355dfc)